ceno_zkvm/examples/fibonacci_elf.rs

```diff
@@ -209,7 +230,7 @@ fn main() {
             MemFinalRecord {
                 addr: rec.addr,
                 value: vm.peek_memory(vma),
-                cycle: *final_access.get(&vma).unwrap_or(&0),
+                cycle: final_access.get(&vma).copied().unwrap_or_default(),
```

Comment:
0 is the intended value. No reason to hide it.

Reply:
I'm fine either way here.